prac_06/languages.py

```diff
@@ -0,0 +1,26 @@
+"""
+Creating a new programming language class.
+Estimated time: 1 hour
+Actual Completion time: 45 minutes
+"""
+
+from prac_06.programming_language import ProgrammingLanguage
+
+
+def main():
+    python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+    ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+    visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+
+    languages = [python, ruby, visual_basic]
+
+    for language in languages:
+        print(language)
+
+    print("The dynamically typed languages are:")
+    for language in languages:
```

Comment:
Although it makes it less readable and only saves 1 line, consider adding a list comprehension here.